content/articles/create-screen-recorder-in-javascript/index.md

```diff
@@ -0,0 +1,219 @@
+title: Section Engineering Education
+
+description: Resources created by engineers for engineers
+
```

Comment:
I assume this is the front matter. This should not worry you. The front matter will always be updated by the peer reviewer so it is not necessary for you to do it by yourself.

Reply:
i have changed this